packages/server/src/integrations/snowflake.ts

```diff
@@ -15,7 +15,7 @@ const SCHEMA: Integration = {
   description:
     "Snowflake is a solution for data warehousing, data lakes, data engineering, data science, data application development, and securely sharing and consuming shared data.",
   friendlyName: "Snowflake",
-  type: "Relational",
+  type: "Non-relational",
```

Comment:
I know it's not a data source plus, but we could make it one without a huge amount of effort. Best to keep it in `relational` for now.

Reply:
Changed back! Updated the screenshot in the OP too.